packages/spark-core/_spark-core.scss

```diff
@@ -56,11 +56,10 @@
 @import './components/buttons';
 @import './components/datepicker';
 @import './components/dividers';
-@import './components/hamburger';
+@import './components/menu';
```

Comment:
Switched hamburger to menu to describe its purpose and its more consistent with how we're referring to it in the designs.